<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateToolsDlg</class>
 <widget class="QDialog" name="CreateToolsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建刀具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>刀具列表</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listWidget">
       <property name="currentRow">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>新建</string>
         </property>
         <property name="icon">
          <iconset resource="Resources/Custom.qrc">
           <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>编辑</string>
         </property>
         <property name="icon">
          <iconset resource="Resources/Custom.qrc">
           <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>删除</string>
         </property>
         <property name="icon">
          <iconset resource="Resources/Custom.qrc">
           <normaloff>:/icons/Remove.png</normaloff>:/icons/Remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources/Custom.qrc"/>
 </resources>
 <connections/>
</ui>
